apps/cli-gui/src/assets/core/reset.scss

```diff
@@ -0,0 +1,5 @@
+*,
```

Comment:
Is it really necessary? Did it actually fixed something? (I'm referring to the whole file)

Reply:
@ronnetzer yes otherwise we would have a visible scroll in the menu